2024/01/22 17:24:01 [error] 176#176: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 108.162.227.130, server: apiunified.snc.id, request: "GET /v1 HTTP/1.1", upstream: "http://34.101.37.32:8080/v1", host: "apiunified.snc.id"
2024/01/22 17:24:01 [error] 176#176: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 108.162.227.105, server: apiunified.snc.id, request: "GET /favicon.ico HTTP/1.1", upstream: "http://34.101.37.32:8080/favicon.ico", host: "apiunified.snc.id", referrer: "https://apiunified.snc.id/v1"
